From 40ed2c409d2276eae8de3071b67859ad705b450f Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Tue, 21 Mar 2023 11:48:31 +0530
Subject: [PATCH] Updating the clang version to 17.0.2

Updating based on Android U
Also changing path from lib64 to lib in latest Clang version

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 core/definitions.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/definitions.mk b/core/definitions.mk
index 4636a83..8d1d0f4 100755
--- a/core/definitions.mk
+++ b/core/definitions.mk
@@ -88,9 +88,9 @@ IAFW_CC := $(IAFW_TOOLS_CLANG_PREFIX)/clang
 IAFW_OBJCOPY := $(IAFW_TOOLS_GCC_PREFIX)objcopy$(HOST_EXECUTABLE_SUFFIX)
 EFI_OBJCOPY := $(IAFW_OBJCOPY)
 ifeq ($(TARGET_IAFW_ARCH),x86_64)
-IAFW_LIBCLANG := $(IAFW_TOOLCHAIN_CLANG_ROOT)/../lib64/clang/14.0.6/lib/linux/libclang_rt.builtins-x86_64-android.a
+IAFW_LIBCLANG := $(IAFW_TOOLCHAIN_CLANG_ROOT)/../lib/clang/17.0.2/lib/linux/libclang_rt.builtins-x86_64-android.a
 else
-IAFW_LIBCLANG := $(IAFW_TOOLCHAIN_CLANG_ROOT)/../lib64/clang/14.0.6/lib/linux/libclang_rt.builtins-i686-android.a
+IAFW_LIBCLANG := $(IAFW_TOOLCHAIN_CLANG_ROOT)/../lib/clang/17.0.2/lib/linux/libclang_rt.builtins-i686-android.a
 endif
 
 # Transformation definitions, ala build system's definitions.mk
-- 
2.39.2

